src/logger/index.ts

```diff
@@ -65,9 +66,23 @@ if (config.logger.loggly.token) {
   });
 }
 
+if (config.google.credentials) {
+  const resource = {
+    type: 'generic_node',
+    labels: {
+      node_id: config.hostname,
+      location: '',
+      namespace: '',
```

Comment:
Are these left empty intentionally?

Reply:
Yeah.  I'll add a comment explaining it, but all stackdriver logs are for a resource of some type, and each type has certain labels that are always present, but default to empty strings.  For `generic_node` resources, these are `project` (which is provided by the application credentials), `node_id`, `location` (ie region), and `namespace`.  I didn't have any ideas of what values to give the location and namespace, and didn't think we need them fully configurable right now, but I thought making their default values explicit was more informative.